code/modules/mob/living/carbon/xenomorph/xenoprocs.dm

```diff
@@ -636,9 +636,12 @@
 		return TRUE
 	return FALSE
 
-/mob/living/carbon/human/can_sting()
-	if(species?.species_flags & IS_SYNTHETIC)
-		return FALSE
```

Comment:
this is unnecessary, only humans should be able to be stung

Reply:
Why not future proof? I don't really care too much, but I feel it's best to make things as applicable as possible unless there's some performative or gameplay reason not to do so.

Reply:
Xenos can't process chems

Reply:
I can certainly exempt them